drivers/hmis/spec/requests/hmis/ce/submit_ce_referral_step_spec.rb

```diff
@@ -165,6 +179,23 @@ def build_input(contact_date: nil, client_accepted: nil)
             end.to change(step, :status).to('completed')
           end
         end
+
+        context 'when submitting this step enables another step that the user is participating in' do
+          before do
+            step.assignments.create!(user: hmis_user) # Assign the user to the current task so they have permission
+            referral.participants.create(swimlane: provider_swimlane, user: hmis_user) # Assign the current user as a participant on the next task's swimlane
+          end
+
+          it 'submits the step' do
```

Comment:
Suggest updating test name to describe the thing being tested here (that isn't tested in other cases?)
```suggestion
          it 'resolves the next available step' do
```